Classes/Controller/CalendarController.php

```diff
@@ -281,6 +281,27 @@ public function yearsAction()
                     'title' => $year['title']
                 ];
             }
+            // create an array that includes years without issues
+            if($this->settings['showEmptyYears']){
```

Comment:
We need to check configuration options with `empty()`, because when updating from an older version, new options don't always get set accordingly, so `$this->settings['showEmptyYears']` may be unset. `empty()` doesn't issue a warning when given an unset variable.

```suggestion
            if (!empty($this->settings['showEmptyYears'])){
```

Reply:
Thank you for the explaination, will keep it in mind next time.